integrated_channels/sap_success_factors/client.py

```diff
@@ -28,6 +28,8 @@ class SAPSuccessFactorsAPIClient(IntegratedChannelApiClient):  # pylint: disable
 
     SESSION_TIMEOUT = 5
 
+    GENERIC_COURSE_COMPLETION_PATH = 'learning/odatav4/public/admin/learningevent-service/v1/OCNLearningEvents'
```

Comment:
any particular reason we are specifying this here and not in `global_sap_configuration`?

Reply:
1. The global config as a whole just feels like an unnecessary object. To store in a database allows us to track changes, but if API endpoints change, it's also extremely likely we'd need to change our code to work with those anyways. Thus I don't really want API endpoints editable in Django Admin. We could expose it in the admin screen but without being able to be edited, but we should just add that to any documentation ECS uses.
2. We decided against the global configs in all of our new integrated channels (see above). I'd love to get rid of the existing endpoints, but don't really go through the refactor at this moment. If this whole thing works out well and we end up wanting to remove the existing course completion url, I'll probably refactor at that point.

Reply:
Oh I forgot reason 3 and sort of 4: by making it a constant we remove an unnecessary (IMO) database call. And (4) a database table that is only ever expected to have a single row just should not exist. I mean our table doesn't because it also acts as a history/audit table, but that has also caused problems in other instances and I'm also not a fan of it.

Reply:
Haha, all good and I agree with all points made -- thanks!